Uplift/Extensions/Date+Shared.swift

```diff
@@ -10,30 +10,34 @@ import Foundation
 
 extension Date {
     static let secondsPerDay = 86400.0
-    
+
+    static var tomorrow: Date {
+        return Calendar.current.date(byAdding: .day, value: 1, to: Date())!
+    }
+
     /// Format: "Today, April 10th 2020"
     static public func getLongDateStringFromDate(date: Date) -> String {
         let weekdayFormatter = DateFormatter()
         weekdayFormatter.dateFormat = "E"
-        
+
         let monthDayFormatter = DateFormatter()
         monthDayFormatter.dateFormat = "MMM d"
-        
+
         let yearFormatter = DateFormatter()
         yearFormatter.dateFormat = "yyyy"
-        
+
         var dayPostfix = "th"
         switch Calendar.current.component(.day, from: date) {
-            case 1:
-                dayPostfix = "st"
-            case 2:
-                dayPostfix = "nd"
-            case 3:
-                dayPostfix = "rd"
-            default:
-                dayPostfix = "th"
+        case 1:
+            dayPostfix = "st"
+        case 2:
+            dayPostfix = "nd"
+        case 3:
+            dayPostfix = "rd"
+        default:
+            dayPostfix = "th"
         }
-        
+
         if date.isToday() {
             return "Today, \(monthDayFormatter.string(from: date))\(dayPostfix) \(yearFormatter.string(from: date))"
```

Comment:
Would it be better here to just have two different formatters (one for if the date is today, and one for if it was yesterday) but have day, month, and year all formatted together?

Reply:
Maybe? That's not something I really added or changed in this PR tho the edits just show up because I was updating the styling to appease the linter.